_data/ZacharySeebeck2020/ZacharySeebeck2020.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Zachary Seebeck
+institution: Bay City Central High School
+profile_pic: zach.jpg
+quote: "Learn as much as you can when given the time. You'll never regret it."
```

Comment:
Remove double quotes from begin and end.

Reply:
It should be fixed!